.golangci-maximal.yml

```diff
@@ -0,0 +1,182 @@
+linters:
+  disable-all: true
+  enable:
+    - copyloopvar
+    - cyclop
+    - depguard
+    - dupword
+    - errcheck
+    - errname
+    - errorlint
+    - exhaustruct
+    - exptostd
+    - funlen
+    - gci
+    - gocheckcompilerdirectives
+    - gocognit
+    - gocritic
+    - godot
+    - goimports
+    - govet
+    - iface
+    - importas
+    - ineffassign
+    - intrange
+    - maintidx
+    - misspell
+    - nestif
+    - nilnesserr
+    - nolintlint
+    - predeclared
+    - recvcheck
+    - revive
+    - staticcheck
+    - testifylint
+    - unconvert
+    - unparam
+    - unused
+    - usestdlibvars
+    - usetesting
+  fast: false
+
+linters-settings:
+  depguard:
+    rules:
+      # This is just an example of a rule you could create, though 
+      forbid-some-pacakge:
+        list-mode: Lax
+        files:
+          - "**/*.go"
+        deny:
+          pkg: "github.com/some/package"
+          desc: "Don't use some/package. It's got issues."
```

Comment:
This should be a list of objects. Also, consider including a useful set of defaults.

```yml
deny:
  - pkg: "github.com/pkg/errors"
    desc: "Use the stdlib errors package instead."
  - pkg: "golang.org/x/crypto/pbkdf2"
    desc: "Use the stdlib crypto/pbkdf2 package instead."
  - pkg: "golang.org/x/crypto/hkdf"
    desc: "Use the stdlib crypto/hkdf package instead."
  - pkg: "golang.org/x/crypto/sha3"
    desc: "Use the stdlib crypto/sha3 package instead."
  - pkg: "github.com/FiloSottile/mlkem768"
    desc: "Use the stdlib crypto/mlkem package instead."
```